src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -18521,6 +18733,31 @@ public class C : I0
         }
 
         [Fact]
+        [WorkItem(20528, "https://github.com/dotnet/roslyn/issues/20528")]
+        public void ExplicitInterfaceImplementationWithNoNames()
+        {
+            var source = @"
+public interface I0
+{
+    void M1((int, int b) x);
+    void M2((int a, int b) x);
+    (int, int b) MR1();
+    (int a, int b) MR2();
+}
+public class C : I0
+{
+    void I0.M1((int, int) z) { }
+    void I0.M2((int, int) z) { }
+    (int, int) I0.MR1() { return (1, 2); }
+    (int, int) I0.MR2() { return (1, 2); }
+}
```

Comment:
Consider adding an explicit implementation of a property if not tested elsewhere.